.github/actions/start-localstack/action.yml

```diff
@@ -0,0 +1,31 @@
+name: Start LocalStack
```

Comment:
GH action to start LocalStack in the docker

Reply:
God bless you